vultr/provider_test.go

```diff
@@ -1,16 +1,35 @@
 package vultr
 
 import (
+	"os"
 	"testing"
 
 	"github.com/hashicorp/terraform/helper/schema"
 	"github.com/hashicorp/terraform/terraform"
 )
 
-var _ terraform.ResourceProvider = Provider()
+var testAccProviders map[string]terraform.ResourceProvider
+var testAccProvider *schema.Provider
+
+func init() {
+	testAccProvider = Provider().(*schema.Provider)
+	testAccProviders = map[string]terraform.ResourceProvider{
+		"vultr": testAccProvider,
+	}
+}
 
 func TestProvider(t *testing.T) {
 	if err := Provider().(*schema.Provider).InternalValidate(); err != nil {
-		t.Fatalf("expected provider to validate: %v", err)
+		t.Fatalf("err: %s", err)
```

Comment:
Is it convention in terraform providers to not have extra context? Otherwise it is nice to add, especially because wrapping the error with `err:` is quite unhelpful as we already know it was an error.